Dockerfile

```diff
@@ -1,71 +1,50 @@
-# Build compilation image
-FROM ruby:3.1.4-alpine as builder
+FROM ruby:3.1.4-alpine3.18 as base
 
-# The application runs from /app
 WORKDIR /app
 
-# Add the timezone as it's not configured by default in Alpine
+# Add the timezone as it’s not configured by default in Alpine
 RUN apk add --update --no-cache tzdata && \
   cp /usr/share/zoneinfo/Europe/London /etc/localtime && \
   echo "Europe/London" > /etc/timezone
 
-# build-base: complication tools for bundle
+# build-base: compilation tools for bundle
 # yarn: node package manager
 # postgresql-dev: postgres driver and libraries
-RUN apk add --no-cache build-base yarn postgresql-dev git
+RUN apk add --no-cache build-base yarn postgresql-dev git bash
```

Comment:
Useful to have bash on the image when you want to debug things on it - but not essential

Reply:
We think we need to install git due to a gem called "overcommit". Seems like it's being used for git hooks. Do you know if we still use it?

Reply:
We will wait until the main CORE codebase has transitioned to Yarn 2 before we use it here.

Reply:
The versions for all these packages have been fixed now, see updated file